deploy/dashboards/quarkus.yaml

```diff
@@ -1,41 +1,26 @@
-apiVersion: "monitoring.kiali.io/v1alpha1"
+apiVersion: "monitoring.kiali.io/v1alpha2"
 kind: MonitoringDashboard
 metadata:
   name: quarkus
 spec:
   title: Quarkus Metrics
   runtime: Quarkus
-  charts:
-    - name: "Thread count"
+  items:
+  - chart:
+      name: "Thread count"
       spans: 4
       metricName: "vendor:thread_count"
       dataType: "raw"
-    - name: "Used heap"
+  - chart:
+      name: "Used heap"
       unit: "bytes"
       spans: 4
       metricName: "vendor:memory_heap_usage_bytes"
       dataType: "raw"
-    - name: "Used non-heap"
+  - chart:
+      name: "Used non-heap"
       unit: "bytes"
       spans: 4
       metricName: "vendor:memory_non_heap_usage_bytes"
       dataType: "raw"
-
-    - name: "Young GC time"
-      unit: "seconds"
-      spans: 3
-      metricName: "base:gc_young_generation_scavenger_time_seconds"
-      dataType: "raw"
-    - name: "Young GC count"
-      spans: 3
-      metricName: "base:gc_young_generation_scavenger_count"
-      dataType: "raw"
-    - name: "Total GC time"
-      unit: "seconds"
-      spans: 3
-      metricName: "base:gc_complete_scavenger_time_seconds"
-      dataType: "raw"
-    - name: "Total GC count"
-      spans: 3
-      metricName: "base:gc_complete_scavenger_count"
-      dataType: "raw"
+  - include: "microprofile-x.y"
```

Comment:
@pilhuhn can you give me the version of microprofile that you used to create this dashboard, so that I can update its name?